package com.kh.example.chap01_exception.run;
import com.kh.example.chap01_exception.controller.ExceptionController;

public class Run {
	public static void main(String[] args) {
		

		ExceptionController ec = new ExceptionController();
		ec.method1();
	//	ec.method2();
	//	ec.method3();
		System.out.println("정상적으로 종료됨...");
	}	
}
